database/migrations/model/Version20230227172459.php

```diff
@@ -0,0 +1,42 @@
+<?php namespace Database\Migrations\Model;
```

Comment:
this migration is not correct lets do what this
https://github.com/OpenStackweb/summit-api/blob/f1d97e47f3f0efca97ad7ddf11224fdc14418cde/database/migrations/model/Version20230227172459.php
and this
https://github.com/OpenStackweb/summit-api/blob/main/database/migrations/model/Version20230125230903.php